server/core/store_stats.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"math"
+	"sync"
+
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/tikv/pd/pkg/movingaverage"
+)
+
+type storeStats struct {
+	mu       sync.RWMutex
+	rawStats *pdpb.StoreStats
+	// smooth available size and record deviation to calculate region score.
+	avgAvailable             *movingaverage.HMA
+	maxAvailableDeviation    *movingaverage.MaxFilter
+	avgMaxAvailableDeviation *movingaverage.HMA
```

Comment:
I think that we should add some comments for them

Reply:
ok